Dockerfile

```diff
@@ -1,12 +1,8 @@
-FROM golang:1.16-alpine3.13 AS builder
-WORKDIR /go/src/github.com/odpf/stencil
-COPY . .
-RUN apk add make bash
-RUN make dist
-
 FROM alpine:3.13
-RUN apk --no-cache add ca-certificates bash
-WORKDIR /root/
+
+RUN apk add --no-cache ca-certificates && update-ca-certificates
+
+COPY stencil /usr/bin/stencil
```

Comment:
This dockerImage assumes executable available already. This creates dependency for building docker image. This impacts local image dev build right? We need to generate executable for alpine separately for docker build to work.